phpseclib/Crypt/RSA.php

```diff
@@ -286,7 +286,7 @@ public static function setOpenSSLConfigPath(string $val): void
      *
      * The public key can be extracted from the private key
      *
-     * @return RSA
+     * @return RSA\PrivateKey
      */
     public static function createKey(int $bits = 2048)
```

Comment:
Add the return type directly:
```suggestion
    public static function createKey(int $bits = 2048): PrivateKey
```